src/locales/en.json

```diff
@@ -292,9 +295,9 @@
     "edit_markup_action": "Edit markup",
     "edit_rate": "Edit rate",
     "empty_state": {
-      "desc": "Create a cost model to start calculating your hybrid cloud costs using custom price lists, markups, or both. Click on the \"Create cost model\" button below to begin the journey.",
+      "desc": "Create a cost model to start calculating your hybrid cloud costs using custom price lists, markups, or both. Click on the button below to begin the journey.",
       "primary_action": "Create cost model",
-      "title": "What is your hybrid cloud costing you?"
+      "title": "At what cost is your hybrid cloud costing you?"
```

Comment:
I realize that the use of the term "cost" is repeated. So just don't do this change, revert back to the old title.